src/main/java/leets/memoriessquare/domain/photo/presentation/PhotoController.java

```diff
@@ -82,4 +80,16 @@ public CountPhotoResponse getCountByDate(@AuthenticationPrincipal OAuthDetails a
         LocalDate now = LocalDate.now();
         return new CountPhotoResponse(countPhotoByDate.execute(auth.getId(), now.getYear(), now.getMonthValue()));
     }
+
+    @Operation(summary = "사용자별 사진 가져오기", description = "특정 사용자의 모든 사진을 조회합니다.")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200"),
+            @ApiResponse(responseCode = "401", content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
+            @ApiResponse(responseCode = "403", content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
+            @ApiResponse(responseCode = "500", content = @Content(schema = @Schema(implementation = ErrorResponse.class)))
+    })
+    @GetMapping("/user")
```

Comment:
```suggestion
    @GetMapping()
```
특정 조건 없이 전체 조회니까 `/photo` endpoint으로만 쓰는 게 좋을 거 같아요